java/app/build.gradle

```diff
@@ -5,15 +5,16 @@ Properties properties = new Properties()
 properties.load(project.rootProject.file('local.properties').newDataInputStream())
 
 fullstory {
-    org properties.getProperty('org') // your ORG_ID in local.properties, or set it here directly
-    if(org == null) throw new GradleException("ORG ID can not be null")
+    // add your "org" in local.properties, or set it here directly
+    org properties.getProperty('org')
+    if(org == null) throw new GradleException('ORG ID is empty, will not record! Please set org in build.gradle(app) or local.properties!')
```

Comment:
- This could be an exception or warning, but gradle warnings are not loud enough so a lot of people don't look at them. -- I'd rather error out on sync than having people run the project and not have the recording work but up for debate
- same thing with if people tries to build without enabling debug variants